README.md

```diff
@@ -44,7 +44,7 @@ The `knapsack_pro` gem supports all CIs and the following test runners:
 
 ## Requirements
 
-`>= Ruby 2.1.0`
+`>= Ruby 3.0.0`
```

Comment:
It should work with Ruby 2.7+ as well but I'm not sure if it's a good idea to mention it here. Ruby 2.7 is deprecated.